taipy/gui_core/_context.py

```diff
@@ -174,7 +174,14 @@ def submission_status_callback(self, submission_id: t.Optional[str]):
                             self.gui._call_user_callback(
                                 client_id,
                                 submission_name,
-                                [core_get(submission.entity_id), {"submission_status": new_status.name}],
+                                [
+                                    core_get(submission.id),
+                                    {
+                                        "submission_status": new_status.name,
+                                        "submittable_entity_id": core_get(submission.entity_id),
+                                        **(event.metadata if event else {}),
```

Comment:
any possibility that event.metadata holds the other 2 keys ?
why not put it first and then override with the other 2 keys ?

Reply:
I suppose you need to propagate the change to the [viselements.json](https://vscode.dev/github/Avaiga/taipy/blob/develop/taipy/gui_core/viselements.json#L195)

Reply:
"any possibility that event.metadata holds the other 2 keys ?" -> hm I don't think so :/

Reply:
👍